src/util/pelz_io.c

```diff
@@ -327,45 +331,109 @@ int read_pipe(char *msg)
   int len;
   char opt;
   charbuf key_id;
+  charbuf path;
 
   if (memcmp(msg, "pelz -", 6) == 0)
   {
     opt = msg[6];
     pelz_log(LOG_DEBUG, "Pipe message: %d, %c, %s", strlen(msg), opt,  msg);
     switch (opt)
     {
-    case 't':
-      key_table_destroy(eid, &ret);
-      if (ret)
-      {
-        pelz_log(LOG_ERR, "Key Table Destroy Failure");
-        return (1);
-      }
-      pelz_log(LOG_INFO, "Key Table Destroyed");
-      key_table_init(eid, &ret);
-      if (ret)
-      {
-        pelz_log(LOG_ERR, "Key Table Init Failure");
-        return (1);
-      }
-      pelz_log(LOG_INFO, "Key Table Re-Initialized");
-      return 0;
-    case 'w':
-      len = strcspn(msg, "\n");
-      key_id = new_charbuf(len - 8); //the number 8 is used because it the number of chars in "pelz -& "
-      memcpy(key_id.chars, &msg[8], (key_id.len));
-      key_table_delete(eid, &ret, key_id);
-      if(ret)
-        pelz_log(LOG_ERR, "Delete Key ID from Key Table Failure: %.*s", (int) key_id.len, key_id.chars);
-      else
-        pelz_log(LOG_INFO, "Delete Key ID form Key Table: %.*s", (int) key_id.len, key_id.chars);
-      return 0;
     case 'e':
       if (unlink(PELZFIFO) == 0)
         pelz_log(LOG_INFO, "Pipe deleted successfully");
       else
         pelz_log(LOG_INFO, "Failed to delete the pipe");
       return 1;
+    case 'l':
```

Comment:
Because this pipe is getting pretty convoluted, can you provide a description in the comments before the parser describing what the options on the pipe mean? Something similar to the 'usage' block but just as comments to assist with parsing the code.

Reply:
Add descriptions